core/controllers/question_player.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Controllers for the questions player."""
+
+from core.controllers import acl_decorators
+from core.controllers import base
+from core.domain import skill_domain
+from core.domain import skill_services
+import utils
+
+
+class SkillMasteryDataHandler(base.BaseHandler):
```

Comment:
Sorry for catching this late - Can we rename this file as skill_mastery instead of question_player? This isn't doing any work that's specific to the question player

Reply:
Done.